### DISCUSSION TITLE
[Feature] Extend Gallery Support

### DISCUSSION THREAD

**Discussion by bpawnzZ** [immich-app/immich#11244] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I think it would be really cool to have the ability to attach volumes that contain photos, via docker.  And then be able to view those in albums or what not inside immich(some sort of way to bulk import and process and viewi in albums).  I love having my mobile photos automatically uploaded.  But it would be great to be able to drop volume of say 10k stable diffusion generations into Immich and have a way to look through them in an album type view.  

Just and idea. but mobile uploads and imported albums should deff be seperate and not in same feed. I have been using immich for a few weerks not and its deff growing on me..

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
https://immich.app/docs/features/libraries/
https://immich.app/docs/features/command-line-interface/
